datamade/la-metro-councilmatic
# Revert "Remove Solr dependency on events page (#424)"

## Overview

This PR reverts commit 1b46b8f7b2d693df2ebca8dd83eb23eb1feb21c8, because the feature is not behaving as expected: It is showing a minutes link for an event that does not yet have a minutes link on production: https://lametro.datamade.us/event/regular-board-meeting-94b40dbd5d08/. The linked file does not exist on the staging or production sites: https://boardagendas.metro.net/board-report/2019-0200/. 

### Checklist


### Notes

N/A

## Testing Instructions

 * Pull down changes
* Navigate to the Events page and confirm that it works
* Navigate to a board report detail page and confirm that it works
